Formula/w/wownero.rb

```diff
@@ -50,7 +49,35 @@ class Wownero < Formula
 
   conflicts_with "monero", because: "both install a wallet2_api.h header"
 
+  # Currently not compatible with boost 1.86.0 even with `monero` fix
+  # https://github.com/monero-project/monero/commit/ed955bf751e304569cd4c04f558360154e19610e
+  resource "boost" do
+    url "https://github.com/boostorg/boost/releases/download/boost-1.85.0/boost-1.85.0-b2-nodocs.tar.xz"
+    sha256 "09f0628bded81d20b0145b30925d7d7492fd99583671586525d5d66d4c28266a"
+  end
```

Comment:
Is there an upstream issue for this? Probably doesn't hurt to keep `boost@1.85` for a bit longer.

Reply:
No. Though, I wasn't able to create an account upstream (HTTP 500 errors on sign-in page) - https://git.wownero.com/wownero/wownero/issues

Part of fix requires upstream syncing with `monero` while rest requires fixes in their own code.

`wownero` is the only remaining usage of `boost@1.85` so it essentially determines when we deprecate it.

Reply:
`wownero` has had under 700 installs in the last year. If we want to get rid of `boost@1.85` let's just deprecate it.